awx/ui/test/e2e/tests/test-workflow-visualizer.js

```diff
@@ -0,0 +1,53 @@
+import {
+    getInventorySource,
+    getJobTemplate,
+    getProject,
+    getWorkflowTemplate
+} from '../fixtures';
+
+let data;
+const workflowTemplateNavTab = "//at-side-nav-item[contains(@name, 'TEMPLATES')]";
+const workflowSelector = "//a[contains(text(), 'test-actions-workflow-template')]";
+const workflowVisualizerBtn = "//button[contains(@id, 'workflow_job_template_workflow_visualizer_btn')]";
+
+const rootNode = "//*[@id='node-2']";
+const childNode = "";
+const leafNode = "//g[contains(@id, 'node-1')]";
+const project = "//td[contains(text(), 'test-actions-project')]";
+const edgeTypeDropdown = "//span[contains(@id, 'select2-workflow_node_edge-container')]";
+const alwaysDropdown = "//*[@id='select2-workflow_node_edge-result-elm7-always']"
+const successDropdown = "//*[@id='select2-workflow_node_edge-result-veyc-success']"
+const failureDropdown = "//*[@id='select2-workflow_node_edge-result-xitr-failure']"
+
+module.exports = {
+    before: (client, done) => {
+        const resources = [
+            getInventorySource('test-actions'),
+            getJobTemplate('test-actions'),
+            getProject('test-actions'),
+            getWorkflowTemplate('test-actions'),
+        ];
+
+        Promise.all(resources)
+            .then(([source, template, project, workflow]) => {
+                data = { source, template, project, workflow };
+                done();
+            });
+        client
+            .login()
+            .waitForAngular()
+            .resizeWindow(1200, 1000)
```

Comment:
let's avoid doing this midtest. We can launch chrome with a certain window size if it's necessary.

Reply:
Noticed this happens across a ton of the UI tests ~15 times. I've made https://github.com/ansible/awx/issues/2426 to track the issue to be handled en masse.